hydrographic_core_msgs/msg/RawBinaryStamped.msg

```diff
@@ -0,0 +1,8 @@
+# message to contain raw binary data from a sensor.
+
+# this message should be timestamped at the receive time of the first byte of the message.
+
+# the frame_id should be that of the sensor which sent the message
+
```

Comment:
As mentioned in my overall comment, I'd like to add a direction flag, so that we can log both commands sent TO the device as well as data received FROM it. This can be super useful for debugging interaction with a more complicated device, as well as just for logging what your configuration actually was.

WHOI's [RawData.msg](https://bitbucket.org/whoidsl/ds_msgs/src/master/ds_core_msgs/msg/RawData.msg) has these fields, which I'd be inclined to just directly copy:
~~~
int8 DATA_OUT=0 
int8 DATA_IN=1

int8 data_direction
~~~